kyototycoon/README.md

```diff
@@ -26,6 +26,30 @@ The KyotoTycoon check is included in the [Datadog Agent][1] package, so you don'
 
 2. [Restart the Agent][4].
 
+##### Log collection
+
+_Available for Agent versions >6.0_
+
+1. Collecting logs is disabled by default in the Datadog Agent, enable it in your `datadog.yaml` file:
+
+    ```yaml
+    logs_enabled: true
+    ```
+
+2. Add this configuration block to your `kyototycoon.d/conf.yaml` file to start collecting Kyoto Tycoon logs:
+
+    ```yaml
+    logs:
+      - type: file
+        path: /var/data/ktserver.log
+        source: kyototycoon
+        service: "<SERVICE_NAME>"
```

Comment:
How come `service` isn't kyototycoon?

Reply:
Thanks, now supersuded as per Christine's comment - the `service` option here was outdated.